src/syscall.h

```diff
@@ -28,4 +50,108 @@ static inline int32_t yield() {
     return syscall(SYSCALL_yield, 0, 0, 0, 0, 0);
 }
 
-#endif
+// Window System Calls
+
+/**
+ * @brief Creates a window for the process
+ * @details Invokes the window_create function with given parameters and assigns
+ * the resulting window to calling process
+ * 
+ * @param x The x position of the window, relative to its parent
+ * @param y The y position of the window, relative to its parent
+ * @param width The width of the window
+ * @param height The height of the window
+ * @return 0 if a window was created, otherwise error
+ */
+static inline int32_t create_window(int x, int y, int width, int height) {
```

Comment:
It actually makes sense to allow a negative x and y -- imagine if you were creating a window which was uncentered. The sides outside of the parents bound would simply be clipped. Width and height should be positive tho, as the behavior with negative width would be undefined.